sql/src/main/java/io/crate/protocols/postgres/SSLUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.protocols.postgres;
+
+import io.netty.channel.Channel;
+import io.netty.handler.ssl.SslHandler;
+
+import javax.annotation.Nullable;
+import javax.net.ssl.SSLSession;
+
+public final class SSLUtils {
+
+    @Nullable
+    public static SSLSession getSslSession(Channel channel) {
```

Comment:
hm i think it does not belong into the postgres package because it is accessed by http. i would put it one level higher make it package private and rename it to SSL and call the function getSession